changelogs/unreleased/module-package-cfcache-ignore.yml

```diff
@@ -0,0 +1,4 @@
+description: Added __cfcache__ as ignore dir for module packaging warnings
```

Comment:
I think you will have to escape the underscores. A double underscore is interpreted in markdown as bold text.

Reply:
I'll backtick quote it. That should work as well I think (`__test__`) and it would look better anyway.